payment_manual/indico_payment_manual/__init__.py

```diff
@@ -16,6 +16,17 @@
 
 from __future__ import unicode_literals
 
+from indico.core import signals
 from indico.util.i18n import make_bound_gettext
 
 _ = make_bound_gettext('payment_manual')
+
```

Comment:
one more empty line here (a pep8 check should actually point this out); there are two empty lines before class/function definitions